src/dmd/cppmanglewin.d

```diff
@@ -251,6 +251,7 @@ public:
         //printf("visit(TypeVector); is_not_top_type = %d\n", (int)(flags & IS_NOT_TOP_TYPE));
         if (checkTypeSaved(type))
             return;
+        mangleModifier(type);
```

Comment:
Since this is the only change, and although tests are included, this line isn't tested? Does not compute!

Reply:
@WalterBright and yet the tests fail without it!

Reply:
The explanation is simple, coverage tests are not ran on Windows.

Reply:
See the previous build reports. https://dev.azure.com/dlanguage/dmd/_build/results?buildId=17054&view=logs&j=6775e4b2-0d76-5db7-9bcd-015c7821c824&t=1bf40372-f529-586c-b834-12b53d9adf74